packages/paste-website/src/components/tokens-list/NoTokensFound.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import {StaticImage} from 'gatsby-plugin-image';
+import {Box} from '@twilio-paste/box';
+import {Stack} from '@twilio-paste/stack';
+import {Heading} from '@twilio-paste/heading';
+import {Card} from '@twilio-paste/card';
+import {Button} from '@twilio-paste/button';
+import {Text} from '@twilio-paste/text';
+import {Anchor} from '@twilio-paste/anchor';
+
+interface NoTokensFoundProps {
+  onClearSearch: () => void;
+}
+
+export const NoTokensFound: React.FC<NoTokensFoundProps> = ({onClearSearch}) => (
+  <Card data-cy="tokens-empty-state" padding="space150">
+    <Box display="flex" columnGap="space110">
+      <Box flexShrink={0} size="size20">
+        <StaticImage
```

Comment:
this is still showing up blurry for me, not a big deal, but I wonder why

Reply:
I just upped the source to 2x, looks better (I think 😅).

Reply:
Lol I usually make it like 8x just to be safe